platforms/Cross/vm/sqAtomicOps.h

```diff
@@ -164,9 +169,39 @@ AtomicGet(uint64_t *target)
 							: "m" (lo32(value)), "m" (hi32(value)) \
 							: "memory", "eax", "ebx", "ecx", "edx", "cc")
 #  endif /* __SSE2__ */
-#elif defined(_MSC_VER) && (defined (_M_AMD64) || defined(_MX64) || defined(_M_IA64))
```

Comment:
Because the 64bit case is already handled up there around line 72/78 (see `#if IS_64_BIT_ARCH`). This line would never match anyway.